src/cloudant/changes.py

```diff
@@ -27,6 +27,9 @@ def __init__(self, session, url, **kwargs):
         self._last_seq = kwargs.get('since')
         self._continuous = kwargs.get('continuous', False)
         self._end_of_iteration = False
+        self._params = {'feed': 'continuous'}
```

Comment:
Is there a tension with the parameters (`self._continuous`) above? Looks like maybe kwargs could work in a general fashion as Jordan mentioned?